ts/receiver/dataMessage.ts

```diff
@@ -37,14 +37,13 @@ function cleanAttachment(attachment: any) {
 }
 
 function cleanAttachments(decrypted: SignalService.DataMessage) {
-  const { quote, group } = decrypted;
+  const { quote } = decrypted;
 
   // Here we go from binary to string/base64 in all AttachmentPointer digest/key fields
 
-  // we do not care about group.avatar on Session
-  if (group && group.avatar !== null) {
-    group.avatar = null;
-  }
+  // we do not care about group on Session
```

Comment:
maybe we don't care on desktop but Android still uses it?

Reply:
updated with Session Desktop